rtgui/edit.cc

```diff
@@ -747,7 +747,294 @@ void Rectangle::drawToMOChannel(Cairo::RefPtr<Cairo::Context> &cr, unsigned shor
     }
 }
 
-EditSubscriber::EditSubscriber (EditType editType) : ID(EUID_None), editingType(editType), bufferType(BT_SINGLEPLANE_FLOAT), provider(NULL), dragging(false) {}
+void OPIcon::drivenPointToRectangle(rtengine::Coord &pos,
```

Comment:
Declaring function arguments as `const` is mostly about safety: You cannot accidentally modify what you declare constant whereas with a normal function argument, you usually get a copy which you can modify even though that is usually not intended. With references as in this case, it is even worse since any changes made to the object would actually be visible to the caller. The general rule of thumb is to be as free of side effects as possible to make reasoning about the code easier.